test/unit/app/models/test_models.py

```diff
@@ -296,24 +297,21 @@ def test_new_credit_transfer_check_sender_transfer_limits(create_credit_transfer
     WHEN a new credit transfer is created
     THEN check the correct check_sender_transfer_limits apply
     """
-    from server.utils.transaction_limits import LIMITS
+    from server.utils.transfer_limits import LIMITS
     from server.models.kyc_application import KycApplication
     from server.models import token
-    from server.utils.transfer_enums import TransferTypeEnum
 
     # Sempo Level 0 LIMITS (payment only)
     assert create_credit_transfer.check_sender_transfer_limits() == [
         limit for limit in LIMITS
-        if limit.name == 'Sempo Level 0'
-        and str(create_credit_transfer.transfer_type) in limit.applied_to_transfer_types
```

Comment:
reliance on this for building the comparision list was meant that if the check on the transfer type broke somewhere in the codebase, this list broke as well. In one case, this resulted in an empty list on both sides of the equality ie
`create_credit_transfer.check_sender_transfer_limits() -> assety [] == [] ` 
This means we were passing tests even though none of the limits were being applied 😱

Reply:
nice catch